marklogic-client-api/src/test/java/com/marklogic/client/test/rows/ValidateDocTest.java

```diff
@@ -139,46 +204,108 @@ public void testUsingFromParamAndSchematron() {
         Iterator<RowRecord> rows = rowManager.resultRows(plan).iterator();
         while (rows.hasNext()){
             RowRecord rowRecord = rows.next();
+            System.out.println(rowRecord);
             if(rowRecord.getString("rowId")!=null)
                 expectedUris.add(rowRecord.getString("rowId"));
         }
-        // TODO: uncomment the below after https://bugtrack.marklogic.com/58055 is fixed
-        // assertTrue(expectedUris.size() == 1);
+        // TODO : uncomment below after https://bugtrack.marklogic.com/57978 is fixed
```

Comment:
These tests can just verify that the expected number of rows are returned. And by following the pattern above, where only 1 row is expected to be returned, none of these tests will be impacted by bug 57978 (which only affects result sets with 2 or more rows).